activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Remove hard coded table names for all Active Storage database models.
```

Comment:
@jonathanhefner  I'm not sure if you want me to remove the entirety of the changelog entry. or only the part about isolate_namespace

Please if so tell and i would remove it entirely. thanks!

Reply:
Alright let's do this 👍🏼

Reply:
I've changed the CHANGELOG following your suggestion.